src/sourmash/commands.py

```diff
@@ -346,6 +346,16 @@ def index(args):
         error('sparseness must be in range [0.0, 1.0].')
 
     if args.scaled:
+        if args.scaled < 1:
```

Comment:
this code is not tested

Reply:
(that is, it is not executed by any tests)

Reply:
this is in the `index` function and my current tests only test the code in the `search` function.